src/lib/rmt/wizard_ssl_page.rb

```diff
@@ -87,7 +99,15 @@ def next_handler
     alt_names_items = UI.QueryWidget(Id(:alt_common_names), :Items)
     alt_names = alt_names_items.map { |item| item.params[1] }
 
-    @cert_generator.generate(common_name, alt_names)
+    ca_password = UI.QueryWidget(Id(:ca_password), :Value)
```

Comment:
When I run this on SLES15 in `ncurses` mode it asks me to enter the password and it breaks afterwards:
![ca generation](https://schnizle.in/share/rmt-password-cli.png)

Reply:
fixed here https://github.com/SUSE/yast2-rmt/pull/19/files#diff-a2eb66213b89f27bcc57d84a05f78f4eR62

Reply:
I run into the same issue again with your fix. I removed an existing certificate and run the wizard again. After entering the password it fails in the same way.

Reply:
should be fixed now

Reply:
works now for me! :+1: